app/src/components/UI/Molecules/Form Group/AccountSettingsFormGroup.tsx

```diff
@@ -66,13 +86,34 @@ const AccountSettingsFormGroup: React.FC<{
           />
         </Form.Group>
         <Container className={styles.saveBtn}>
-          <Buttons submit color="green" clicked={() => setDisabledForm(true)}>
-            Save
-          </Buttons>
+          {userQuery.isSuccess ? (
+            <Buttons submit color="green" clicked={() => setDisabledForm(true)}>
+              Save
+            </Buttons>
+          ) : null}
         </Container>
       </Form>
     </Container>
   );
 };
 
+const AccountSettingsFormGroup: React.FC<{
```

Comment:
`AccountSettingsFormGroup` was split into a smaller component so that hooks can be called from a conditional (queries can only be called after token is retrieved).